web/crux/src/app.module.ts

```diff
@@ -42,9 +41,10 @@ const imports = [
   StorageModule,
   ConfigModule.forRoot(appConfig),
   EmailModule,
-  PrometheusModule.register({
-    controller: MetricsHttpController,
-  }),
```

Comment:
```
    PrometheusModule.register({
      defaultMetrics: {
        enabled: false,
      },
    }),
``` 

...not working?

Reply:
This won't disable the default `/api/metrics` endpoint, it only removes default metrics (node GC, memory stats, etc.).

Reply:
Okey, thanks for the clarification.